content/en/docs/reference/glossary/service-catalog.md

```diff
@@ -1,18 +0,0 @@
----
-title: Service Catalog
-id: service-catalog
-date: 2018-04-12
-full_link: 
-short_description: >
-  An extension API that enables applications running in Kubernetes clusters to easily use external managed software offerings, such as a datastore service offered by a cloud provider.
-
-aka: 
-tags:
-- extension
----
- An extension API that enables applications running in Kubernetes clusters to easily use external managed software offerings, such as a datastore service offered by a cloud provider.
-
-<!--more--> 
-
-It provides a way to list, provision, and bind with external {{< glossary_tooltip text="Managed Services" term_id="managed-service" >}} from {{< glossary_tooltip text="Service Brokers" term_id="service-broker" >}} without needing detailed knowledge about how those services are created or managed.
```

Comment:
I think I'd leave the glossary entry in place, and reword it to make it clear that Service Catalog is a _former_ extension mechanism for Kubernetes.

Reply:
Ah, yes, that makes sense